### DISCUSSION TITLE
netbox_capirca_plugin

### DISCUSSION THREAD

**Discussion by dover1987** [netbox-community/netbox#7533] [Q&A]
Hi everyone!
After installing the plugin and clicking on 'ACL' button I get this error. Can someone tell me how to fix it?
At the same time API for this plugin works well and entities are created successfuly. 

![image](https://user-images.githubusercontent.com/51654296/136981749-9beebfd0-ece6-4afe-8e30-d9c5607f9344.png)

**Comment by candlerb** [comment]
That looks like the standard error message which is printed when you try to use a plugin which hasn't been updated to be compatible with Netbox 3.0.

Comment out the plugin in configuration.py.  If that fixes the problem, then report it to the plugin maintainer.
